improver/developer_tools/metadata_interpreter.py

```diff
@@ -92,7 +91,7 @@
 SPOT_COORDS = ["spot_index", "latitude", "longitude", "altitude", "wmo_id"]
 UNBLENDED_TIME_COORDS = ["time", "forecast_period", "forecast_reference_time"]
 BLENDED_TIME_COORDS = ["time", "blend_time"]
-LOCAL_TIME_COORDS = ["time", "time_in_local_timezone"]
+LOCAL_TIME_COORDS = ["time"]
```

Comment:
You can delete `LOCAL_TIME_COORDS` entirely I believe.

Reply:
Quite right. I've removed it entirely.